src/components/token-family/TokenFamilyHeader.js

```diff
@@ -49,7 +49,7 @@ const TitleText = styled(TruncatedText).attrs({
   letterSpacing: 'roundedMedium',
   lineHeight: 'normal',
   size: 'large',
-  weight: 'semibold',
+  weight: 'bold',
```

Comment:
can we use `fontWeights.bold`?

Reply:
@osdnk is this a new pattern? I don't think we do this anywhere else, only with stylesheets we used in the past